tests/templates/test_embeddings/test_amplitude.py

```diff
@@ -359,3 +359,49 @@ def test_torch(self, tol):
         res2 = circuit2(features)
 
         assert qml.math.allclose(res, res2, atol=tol, rtol=0)
+
+
+def test_multiple_amplitudes():
```

Comment:
It's fair game to create a separate test class here (just like `TestInterfaces`). :slightly_smiling_face: That way further test cases can also be added as separate methods.